src/spectrecoff/Output.fs

```diff
@@ -65,8 +65,10 @@ let private stringifyLook look =
     stringify look.Color look.BackgroundColor look.Decorations
 
  // Basic output
-let markup style content =
-    $"[{style}]{Markup.Escape content}[/]"
+let markup (style: string) content =
```

Comment:
The type specification `string` can be inferred and is not necessary here I think.

Also, how about leaving this function as it was, and instead doing `let vanilla content = content |> Markup.Escape` in line 85. 
This way, we avoid this round trip and the empty string getting special handling.

Reply:
String can not be inferred, unfortunately, since any .NET type has a `.ToString()` method, and that will implicitly be used in string interpolation. Would be different if we'd be doing string addition/concatenation.

And about 'leaving it as it is', this is just a safe-guard for the case when it is used with empty styles. At first I tried 
`let vanilla content = content |> markup ""` which lead to an exception, so I thought it makes sense to protect that.

Reply:
Fair enough, but are you quite sure about the type inference? :)

![image](https://github.com/EluciusFTW/SpectreCoff/assets/17160319/063c87f5-614c-4a87-9819-c4dea29192e3)

Reply:
You are right, now it does the same for me as well. Yesterday it assumed styler was of type `'a` (hence I added the annotation), but that might have been in the middle of the changes when something else was messed up. Will revert the ugliness.

Reply:
Done.